src/Service/Service.php

```diff
@@ -29,14 +29,14 @@
      *
      * @var HttpClient
      */
-    private $httpClient;
+    protected $httpClient;
```

Comment:
Can you leave them private ? We can use the `request()` method instead in the services